### PR TITLE
Add a (-1)^l factor to the power spectrum invariants

### PR DESCRIPTION
**Pull request by Luthaf**
This make the power spectrum produce the same output as doing a full Clebsch-Gordan product and keeping only the lambda=0 term

----
:books: Documentation preview :books:: https://rascaline--248.org.readthedocs.build/en/248/

### CODE REVIEW

**Review by jwa7 — APPROVED** [review]
Performed the consistency check of the invariant of lambda-SOAP vs Python PowerSpectrum implementation and it passes, as if the CI looks happy it looks good to me!

File: python/rascaline/rascaline/utils/power_spectrum/calculator.py [hunk 1]
```diff
@@ -217,7 +217,8 @@ def compute(
             ell = key[0]
             species_center = key[1]
 
-            factor = 1 / sqrt(2 * ell + 1)
+            factor = (-1) ** ell / sqrt(2 * ell + 1)
```
Comment (PicoCentauri) [inline python/rascaline/rascaline/utils/power_spectrum/calculator.py:220]: I would add the comment from the rust code here as well:

```suggestion
             # For consistency with a full Clebsch-Gordan product we need to add
             # a `-1^l / sqrt(2 l + 1)` factor to the power spectrum invariants
            factor = (-1) ** ell / sqrt(2 * ell + 1)
```

File: rascaline/src/calculators/soap/power_spectrum.rs [hunk 2]
```diff
@@ -375,6 +386,8 @@ impl SoapPowerSpectrum {
 struct SpxPropertiesToCombine<'a> {
     /// value of l
     spherical_harmonics_l: usize,
+    /// normalization factor (-1)^l * sqrt(2 l + 1)
```
Comment (PicoCentauri) [inline rascaline/src/calculators/soap/power_spectrum.rs:389]: use the power of LateX
```suggestion
    /// normalization factor :math:`-1^l  \sqrt{2 l + 1}`
```

File: scripts/clean-python.sh [hunk 3]
```diff
@@ -10,6 +10,7 @@ cd "$ROOT_DIR"
 
 rm -rf dist
```
Comment (PicoCentauri) [inline scripts/clean-python.sh:11]: Ahh there is file to clean the Python build leftovers. Very useful!

**Review by PicoCentauri — CHANGES_REQUESTED** [review]
Some minor docs comment, otherwise I am happy. I like when we break the code.

**Review by PicoCentauri — APPROVED** [review]
Ship it! 🚢

**Pull request merged**